client/question/components/LearnerViewDirective.js

```diff
@@ -204,7 +202,7 @@ tie.directive('learnerView', [function() {
           float: left;
           margin-top: 10px;
         }
-        .night-mode .tie-code-reset, .night-mode .tie-python-primer {
+        .night-mode .tie-code-reset, .night-mode {
```

Comment:
If you're deleting .tie-python-primer should the .night-mode before it be deleted as well?

Reply:
Yeah, that's right. Fixed!